keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/KeelTagSpec.kt

```diff
@@ -34,8 +35,9 @@ import com.netflix.spinnaker.keel.tags.EntityTag
 data class KeelTagSpec(
   val keelId: String,
   val entityRef: EntityRef,
-  val tagState: TagState
-)
+  val tagState: TagState,
+  override val application: String // this is found from entity ref
```

Comment:
feels like for a case where we have the application source object in the constructor we should get rid of application as a constructor arg and move the `val application = entityRef.application` into the constructor itself